Formula/helium-wallet.rb

```diff
@@ -0,0 +1,28 @@
+class HeliumWallet < Formula
+  desc "Rust implementation of a helium wallet CLI"
+  homepage "https://github.com/helium/helium-wallet-rs"
+  url "https://github.com/helium/helium-wallet-rs.git", { using: :git, tag: "v1.6.6", revision: "4becd5fd9f817ecfee00dba3177a64c7fc223bd1" }
+  license "Apache-2.0"
+  head "https://github.com/helium/helium-wallet-rs.git", { using: :git, branch: "master" }
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+    prefix.install "README.md"
+    prefix.install "LICENSE"
+  end
+
+  test do
+    # Currently disabled since `helium-wallet create basic` requires user input.
```

Comment:
Let's simulate that user input then. We really need a test that does more than `--version` or `--help`.

Reply:
Could you elaborate on the best way of doing this? I've seen some people using Expect but I'm unsure if this is the way to do it.

Reply:
You could try using Expect, or use Ruby's PTY module. The latter is probably simpler -- see formulae that do `require "pty"` for examples.

Reply:
Never worked with PTY before but for some reason it won't spawn the process. Currently using the following:
```rb
  test do
    require "pty"

    key = testpath/"test-wallet.key"
    _, w, pid = PTY.spawn(bin/"helium-wallet", "create", "basic", "-o", key)
    sleep 2
    w.write "test-key-password\r"
    sleep 2
    w.write "test-key-password\r"
    sleep 5

    assert_predicate key, :exist?
  end
```